TLM/TLM/Manager/Impl/JunctionRestrictionsManager.cs

```diff
@@ -645,18 +646,23 @@ public class JunctionRestrictionsManager
             return ret;
         }
 
+        private static bool IsNodePedestrianCrossingConfigurable(ref NetNode node) {
+            return (node.m_flags & (NetNode.Flags.Junction | NetNode.Flags.Bend)) != NetNode.Flags.None
```

Comment:
![image](https://user-images.githubusercontent.com/26344691/168624304-5c3136c2-152d-463e-847e-f47b6d93f7a6.png)
at bend nodes pedestrian crossing is configurable. Is this intentional?

Reply:
@Elesbaan70 are bend nodes within the scope of this PR?